overlord/devicestate/firstboot_test.go

```diff
@@ -167,7 +175,7 @@ func checkTrivialSeeding(c *C, tsAll []*state.TaskSet) {
 	c.Check(tasks[0].Kind(), Equals, "mark-seeded")
 }
 
-func (s *firstBoot16Suite) modelHeaders(modelStr string, reqSnaps ...string) map[string]interface{} {
```

Comment:
why not make this a method on firstBoot16BaseTest?

Reply:
It could, but all it needs is a list of strings and doesn't need any of the test suite stuff; no strong opinion but as long as not needed I prefer standalone helpers.

Reply:
sure this is fine to not be a method then